tests/e2e/network/multiple_standard_lb.go

```diff
@@ -0,0 +1,203 @@
+
+	It("should send nodes to correct load balancers by primary vmSet", func() {
+		if os.Getenv(utils.AKSTestCCM) != "" {
+			Skip("This test is not supported for AKS CCM.")
+		}
+
+		var svcIPs []string
+		svcCount := 2
+		for i := 0; i < svcCount; i++ {
+			svcName := fmt.Sprintf("%s-%d", testServiceName, i)
+			svc1 := utils.CreateLoadBalancerServiceManifest(svcName, nil, labels, ns.Name, ports)
+			_, err := cs.CoreV1().Services(ns.Name).Create(context.TODO(), svc1, metav1.CreateOptions{})
+			Expect(err).NotTo(HaveOccurred())
+			ips, err := utils.WaitServiceExposureAndValidateConnectivity(cs, tc.IPFamily, ns.Name, svcName, []string{})
+			Expect(err).NotTo(HaveOccurred())
+			Expect(len(ips)).NotTo(BeZero())
+			svcIPs = append(svcIPs, ips[0])
+		}
+
+		clusterName := os.Getenv("CLUSTER_NAME")
+		lbNameToExpectedVMSetNameMap := map[string]string{
+			clusterName: fmt.Sprintf("%s-vmss-0", clusterName),
+			"lb1":       fmt.Sprintf("%s-vmss-1", clusterName),
+			"lb2":       fmt.Sprintf("%s-vmss-2", clusterName),
+		}
+
+		for _, svcIP := range svcIPs {
+			lb := getAzureLoadBalancerFromPIP(tc, svcIP, tc.GetResourceGroup(), tc.GetResourceGroup())
+			lbName := pointer.StringDeref(lb.Name, "")
+			for _, bp := range *lb.BackendAddressPools {
+				for _, a := range *bp.LoadBalancerBackendAddresses {
+					nodeName := pointer.StringDeref(a.Name, "")
+					expectedVMSSName := lbNameToExpectedVMSetNameMap[lbName]
+					if nodeName == "" || !strings.HasPrefix(nodeName, expectedVMSSName) {
+						Fail(fmt.Sprintf("Node %s is not in the expected VMSS %s on LB %s", nodeName, expectedVMSSName, lbName))
+					}
+				}
+			}
+		}
+	})
+
+	It("should arrange services across load balancers correctly", func() {
+		var svcIPs []string
+		svcCount := 2
+		var svc *v1.Service
+		By(fmt.Sprintf("Creating %d services", svcCount))
+		for i := 0; i < svcCount; i++ {
+			svcName := fmt.Sprintf("%s-%d", testServiceName, i)
+			svc = utils.CreateLoadBalancerServiceManifest(svcName, nil, labels, ns.Name, ports)
+			_, err := cs.CoreV1().Services(ns.Name).Create(context.TODO(), svc, metav1.CreateOptions{})
+			Expect(err).NotTo(HaveOccurred())
+			ips, err := utils.WaitServiceExposureAndValidateConnectivity(cs, tc.IPFamily, ns.Name, svcName, []string{})
+			Expect(err).NotTo(HaveOccurred())
+			Expect(len(ips)).NotTo(BeZero())
+			svcIPs = append(svcIPs, ips[0])
+		}
+
+		By("Checking the load balancer count to equal 2")
+		lbNames := getLBsFromPublicIPs(tc, svcIPs)
+		Expect(len(lbNames)).To(Equal(2))
+
+		By("Creating a service targeting to a new load balancer")
+		l := map[string]string{
+			"app": testServiceName,
+			"a":   "b",
+		}
+		svcWithLabel := utils.CreateLoadBalancerServiceManifest("svc-with-label", nil, l, ns.Name, ports)
+		_, err := cs.CoreV1().Services(ns.Name).Create(context.TODO(), svcWithLabel, metav1.CreateOptions{})
+		Expect(err).NotTo(HaveOccurred())
+		svsWithLabel, err := utils.WaitServiceExposure(cs, ns.Name, "svc-with-label", []string{})
```

Comment:
Could you please add a defer delete service below L156?

Reply:
It will be removed in afterEach.

Reply:
It is possible that deployment deletion in AfterEach fails then the namespace is not deleted.
If the Service is not cleaned up, its related resources like FIP, rules may affect other tests. This may mislead us to believe there're problems with those tests.